### PR TITLE
Bug/fix facebook images on links

### PR DESCRIPTION
**Pull request by Daveawb**
## Description - [Trello ticket](https://trello.com/c/G18faAEA)

Added Facebook Open Graph meta tags in place for all pages
- YLD logo on all pages
- Casestudies use the poster image instead

Does not add a solution for platforms that don't use Open Graph.

## Review template
- [ ] checked that this PR resolves the spec provided from trello / this description
- [ ] ensured that this PR does not introduce any obvious bugs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,9 @@
 In order to access data from contentful make sure that you have an .env file that matches the contentful keys and tokens:
 
 ```
-CONTENTFUL_TOKEN=(see in contentful/settings/API keys)
-CONTENTFUL_SPACE=(see in contentful/settings/API keys)
-MEETUP_KEY=(see in contentful/settings/API keys)
+CONTENTFUL_TOKEN=(see in contentful/settings/API keys "meetup > Content Delivery API - Access Token")
```
Comment (antoniocapelo) [inline README.md]: I don't understand the `meetup` keyword in a Contentful-related env var. But probably I'm missing something :-)

**Comment by antoniocapelo** [comment]
Not sure if this is inside the scope of this card, but if we're adding `open graph` fields, we should probably add a couple more (such as title, url, etc). Also, it would make sense to implement the `twitter card` meta tags so that when sharing on twitter our cases studies, they still look good.

**Note:** I haven't looked into the source code to see if this `head` component already does that.

As an example, here is an excerpt of some meta tags I added on a site I worked previously, if it helps:

```html
  <meta
    data-react-helmet="true"
    property="og:url"
    content="https://site.name"
    prefix="og: http://ogp.me/ns#"
  />
  <meta
    data-react-helmet="true"
    property="og:type"
    content="website"
    prefix="og: http://ogp.me/ns#"
  />
  <meta
    data-react-helmet="true"
    property="og:title"
    content="Page title"
    prefix="og: http://ogp.me/ns#"
  />
  <meta
    data-react-helmet="true"
    property="og:image"
    content="http://site.name/_build/images/share-social-link.bd1caa627c24ada.png"
    prefix="og: http://ogp.me/ns#"
  />
  <meta
    data-react-helmet="true"
    property="og:description"
    content="Website description"
    prefix="og: http://ogp.me/ns#"
  />
  <meta
    data-react-helmet="true"
    property="og:site_name"
    content="Website Title"
    prefix="og: http://ogp.me/ns#"
  />
  <meta
    data-react-helmet="true"
    property="og:locale"
    content="en_US"
    prefix="og: http://ogp.me/ns#"
  />
  <meta
    data-react-helmet="true"
    name="twitter:card"
    content="summary_large_image"
  />
  <meta data-react-helmet="true" name="twitter:site" content="@twitterhandle" />
  <meta
    data-react-helmet="true"
    name="twitter:url"
    content="https://site.name"
  />
  <meta
    data-react-helmet="true"
    name="twitter:title"
    content="MOXY — Software &amp; Design Studio"
  />
  <meta
    data-react-helmet="true"
    name="twitter:description"
    content="Website description"
  />
  <meta
    data-react-helmet="true"
    name="twitter:image"
    content="http://site.name/_build/images/share-social-link.bd1caa627c24ada.png"
  />
``` 
 
Any thoughts welcome :)

File: README.md (hunk 1 above)
Comment (Daveawb) [reply README.md]: Are we talking about the `MEETUP_KEY` itself or the descriptions I've added?

If it's the descriptions, there are two spaces in contentful and each one has two sets of API keys. The API Keys in `yld.io` space are `meetup` and `gatsby`

Comment (antoniocapelo) [reply README.md]: ok se we've got meetup-related keys inside contentful? interesting :-)

**Comment by Daveawb** [comment]
@antoniocapelo I totally agree but we should probably tackle it in a different ticket than this one. I'll add it to Trello.

**Pull request merged**